src/components/typography/text/text.spec.js

```diff
@@ -109,6 +119,9 @@ describe('<Subheadline>', () => {
     it('should have "primary" class', () => {
       expect(wrapper).toContainClass(styles.primary);
     });
+    it('should has `title` prop', () => {
+      expect(wrapper.prop('title')).toBe('tooltip text');
```

Comment:
I would suggest `toHaveProp`? It gives slightly better error messages.

You could (but only if you feel like it) rewrite the whole thing with `react-testing-library`. Absolutely no worries if not though! :)